frontend/src/metabase-types/api/settings.ts

```diff
@@ -180,7 +180,7 @@ export interface Settings {
   "custom-homepage": boolean;
   "custom-homepage-dashboard": number | null;
   "deprecation-notice-version"?: string;
-  "dismissed-custom-dashboard-toast": boolean;
+  "dismissed-custom-dashboard-toast"?: boolean;
```

Comment:
The setting is not present on the page load, it also gets loaded via `/api/session/properties`. This makes a difference for the home page